frontend/app/routes/_gcweb-app.personal-information.home-address.suggested.tsx

```diff
@@ -0,0 +1,154 @@
+
+  return json({ homeAddressInfo, suggestedAddressInfo });
+}
+
+/**
+ * action, save address, either the entered address or the suggested address depending on the radio button choosen...
+ */
+export async function action({ request }: ActionFunctionArgs) {
+  const userId = await userService.getUserId();
+  const userInfo = await userService.getUserInfo(userId);
+  const homeAddressInfo = await getAddressService().getAddressInfo(userId, userInfo?.homeAddress ?? '');
+  console.debug('ACTION SESSION home address :: ' + JSON.stringify(homeAddressInfo, null, 2));
+  /**
+   * TODO
+   * CHANGE THE SOURCE OF THE SUGGESTED ADDRESS TO WHAT WS ADDRESS SERVICE IS RETURNING INSTEAD OF MAILING ADDRESS
+   */
+  const suggestedAddressInfo = await getAddressService().getAddressInfo(userId, userInfo?.mailingAddress ?? '');
+  const formDataRadio = Object.fromEntries(await request.formData());
+  //retrieve selected address, store it in the session and then redirect to the confirm page...
+  console.debug('SELECTED :: ' + formDataRadio.selectedAddress);
+  const session = await sessionService.getSession(request.headers.get('Cookie'));
+  if (formDataRadio.selectedAddress == 'home') {
+    session.set('newHomeAddress', homeAddressInfo);
+  } else {
+    session.set('newHomeAddress', suggestedAddressInfo);
+  }
+  return redirect('/personal-information/home-address/confirm', {
+    headers: {
+      'Set-Cookie': await sessionService.commitSession(session),
+    },
+  });
+}
+
+/**
+ * MAIN
+ * @returns
+ */
+export default function HomeAddressSuggested() {
+  //const actionData = useActionData<typeof action>();
+
+  const { homeAddressInfo, suggestedAddressInfo } = useLoaderData<typeof loader>();
+  const { t } = useTranslation(i18nNamespaces);
+
+  return (
+    <>
+      <Form method="post">
+        <p>{t('personal-information:home-address.suggested.note')}</p>
+        <div className="grid gap-6 md:grid-cols-2">
+          <PersonalInformationSection title={t('personal-information:home-address.suggested.address-entered')} icon="glyphicon-map-marker">
+            {homeAddressInfo && <Address address={homeAddressInfo?.address} city={homeAddressInfo?.city} provinceState={homeAddressInfo?.province} postalZipCode={homeAddressInfo?.postalCode} country={homeAddressInfo?.country} />}
+            {!homeAddressInfo && <p>{t('personal-information:index.no-address-on-file')}</p>}
+          </PersonalInformationSection>
+          <PersonalInformationSection title={t('personal-information:home-address.suggested.address-suggested')} icon="glyphicon-map-marker">
+            {suggestedAddressInfo ? (
+              <Address address={suggestedAddressInfo?.address} city={suggestedAddressInfo?.city} provinceState={suggestedAddressInfo?.province} postalZipCode={suggestedAddressInfo?.postalCode} country={suggestedAddressInfo?.country} />
+            ) : (
+              <p>{t('personal-information:index.no-address-on-file')}</p>
```

Comment:
I understand that the "Address Entered" and "Suggested Address" are currently just the home and mailing address on file, but when we refine this route, make note that we should always have a "Address Entered" and "Suggested Address" and we should never display this line that indicates there is no address on file.